File: app/src/main/java/com/callmangement/model/inventrory/ModelParts.java
package com.callmangement.model.inventrory;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class ModelParts {
    @SerializedName("status") private String status;
    @SerializedName("message") private String message;
    @SerializedName("Parts") private List<ModelPartsList> Parts = new ArrayList();

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<ModelPartsList> getParts() {
        return Parts;
    }

    public void setParts(List<ModelPartsList> parts) {
        Parts = parts;
    }
}

File: app/src/main/java/com/callmangement/model/inventrory/ModelParts.kt
package com.callmangement.model.inventrory

import com.google.gson.annotations.SerializedName

class ModelParts {
    @JvmField
    @SerializedName("status")
    var status: String? = null

    @JvmField
    @SerializedName("message")
    var message: String? = null

    @SerializedName("Parts")
    var parts: List<ModelPartsList>? = ArrayList()
}
